README.md

```diff
@@ -104,6 +104,11 @@ As mentioned above, the `bucketName` is required, but there's much more to confi
+| `maxRetries`                                | `number \| undefined`                                                                                                           | `undefined`                                                                                                                                                                                | The maximum amount of retries to perform for a service request.                                                                                                                                                                                                                                                                                                                                                      |
+| `connectTimeout`                                | `number \| undefined`                                                                                                           | `undefined`                                                                                                                                                                                | The maximum time in milliseconds that the connection phase of the request should be allowed to take. This only limits the connection phase and has no impact once the socket has established a connection.                                                                                                                                                                                                                                                                                                              |
+| `timeout`                                | `number \| undefined`                                                                                                           | `120000`                                                                                                                                                                                | The number of milliseconds to wait before giving up on a connection attempt.                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
What's the difference between `timeout` and `connectTimeout`? One talks about a connection phase and the other about a connection attempt. If `timeout` applies to the request as a whole, could you please make that more clear?

Reply:
It's actually AWS's own explanation for the prop but I agree it's not very clear. I reworded it a bit, is it more clear now?